### PR TITLE
Restored Killed console message

### PR DESCRIPTION
**Pull request by rjperes**
Added Started console message
Captured possible exception from node process

### CODE REVIEW

File: Microsoft.AspNet.NodeServices/HostingModels/OutOfProcessNodeInstance.cs [hunk 1]
```diff
@@ -103,18 +111,18 @@ public OutOfProcessNodeInstance(string entryPointScript, string commandLineArgum
...
         }
-        
+
         protected virtual void OnBeforeLaunchProcess() {
         }
 
         protected virtual void OnOutputDataReceived(string outputData) {
             Console.WriteLine("[Node] " + outputData);
         }
-
```
Comment (moozzyk) [inline Microsoft.AspNet.NodeServices/HostingModels/OutOfProcessNodeInstance.cs]: The only thing I hate about VS code....

Comment (rjperes) [inline Microsoft.AspNet.NodeServices/HostingModels/OutOfProcessNodeInstance.cs]: You mean, the whitespaces? Arrrgghhh...

Comment (moozzyk) [inline Microsoft.AspNet.NodeServices/HostingModels/OutOfProcessNodeInstance.cs]: Yeah, whitespaces in empty lines

Comment (rjperes) [inline Microsoft.AspNet.NodeServices/HostingModels/OutOfProcessNodeInstance.cs]: If I get rid of them, will you merge? :-)

Comment (moozzyk) [inline Microsoft.AspNet.NodeServices/HostingModels/OutOfProcessNodeInstance.cs]: It was just a random comment, sorry. I only saw a demo of this thing during the MVP summit and it was very cool so I got interested. @SteveSanderson needs to look at it I think.

Comment (rjperes) [inline Microsoft.AspNet.NodeServices/HostingModels/OutOfProcessNodeInstance.cs]: Yeah, he's right here with me! :-)

Comment (moozzyk) [inline Microsoft.AspNet.NodeServices/HostingModels/OutOfProcessNodeInstance.cs]: bug him! :smile:

**Comment by SteveSandersonMS** [comment]
Thanks! I've merged some of the changes here (the better logging of Node startup failures, and the whitespace fixes). I didn't merge the console messages that were being issued on successful Node process startup or shutdown, since we don't want to produce extra log messages if everything is OK.

**Pull request merged**